### ISSUE TITLE
abnormal umbilical artery dopplers in pregnancy

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7006]
**Preferred term label:**
abnormal umbilical artery dopplers in pregnancy

**Synonyms**
elevated umbilical artery dopplers
absent end diastolic flow in umbilical artery dopplers
reversed end diastolic flow in umbilical artery dopplers

**Definition (free text, please give PubMed ID)**
abnormal waveform of the umbilical artery dopplers during pregnancy

**Parent term (use hpo.jax.org/app)**
no existing parent term that I can find

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
fetal growth restriction
placental insufficiency
potentially associated with a large number of fetal genetic disorders (although not always reliably present when a fetal genetic disorder is diagnosed)

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by patt1morgan** [comment]
@jokemuys, @carlotarodo, @JeroenBreckpot.  Greetings!  This is another new term request we have received for the HPO.  When you have the time, could you please review and comment..
Thanks again for all your input...

**Comment by carlotarodo** [comment]
Doppler is a name, it can't be plural and it should be written with capital letters. The term "dopplers" is incorrect.

Don't agree it's potentially related with genetic disorders.

-----

Preferred term label:
abnormal umbilical artery **Doppler** pattern in pregnancy

Synonyms
Not correct: "elevated umbilical artery dopplers" 
Umbilical artery pulsatility index above the 95th centile
Absent end-diastolic flow in umbilical artery Doppler
Reversed end-diastolic flow in umbilical artery Doppler

Definition (free text, please give PubMed ID)
the abnormal waveform of the umbilical artery Doppler during pregnancy

Diseases characterized by this term? (e.g. Orphanet or OMIM number)
fetal growth restriction
placental insufficiency
NO! "potentially associated with a large number of fetal genetic disorders (although not always reliably present when a fetal genetic disorder is diagnosed)"

**Comment by pnrobinson** [comment]
@sparkste  I am adding the parent term. I think there are lots of child terms we could consider but it would be better to revisit this in a zoom meeting later on.
